metacat-main/src/main/java/com/netflix/metacat/main/api/RequestWrapper.java

```diff
@@ -87,7 +98,11 @@ public RequestWrapper(@NotNull @NonNull final Registry registry) {
      */
     public QualifiedName qualifyName(final Supplier<QualifiedName> nameSupplier) {
         try {
-            return nameSupplier.get();
+            final QualifiedName name = nameSupplier.get();
+            if (config.isTableAliasEnabled() && name.isTableDefinition()) {
```

Comment:
Use name.getType() instead of name.isTableDefinition.

Reply:
Will do.